src/Config/ConfigInterface.php

```diff
@@ -1,54 +1,52 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Laminas\OAuth\Config;
 
 use Laminas\OAuth\Token\TokenInterface;
 
-/**
- * @category   Laminas
- * @package    Laminas_OAuth
- */
 interface ConfigInterface
 {
     public function setOptions(array $options);
 
-    public function setConsumerKey($key);
+    public function setConsumerKey(string $key);
 
     public function getConsumerKey();
 
-    public function setConsumerSecret($secret);
+    public function setConsumerSecret(string $secret);
 
     public function getConsumerSecret();
 
-    public function setSignatureMethod($method);
+    public function setSignatureMethod(string $method);
 
     public function getSignatureMethod();
 
-    public function setRequestScheme($scheme);
+    public function setRequestScheme(string $scheme);
 
     public function getRequestScheme();
 
-    public function setVersion($version);
+    public function setVersion(string $version);
 
     public function getVersion();
 
-    public function setCallbackUrl($url);
+    public function setCallbackUrl(string $url);
 
     public function getCallbackUrl();
 
-    public function setRequestTokenUrl($url);
+    public function setRequestTokenUrl(string $url);
 
     public function getRequestTokenUrl();
 
-    public function setRequestMethod($method);
+    public function setRequestMethod(string $method);
 
     public function getRequestMethod();
 
-    public function setAccessTokenUrl($url);
+    public function setAccessTokenUrl(string $url);
 
     public function getAccessTokenUrl();
 
-    public function setUserAuthorizationUrl($url);
+    public function setUserAuthorizationUrl(string $url);
```

Comment:
All parameter type additions should be reverted. Adding parameter types is a BC break - You'll have to add doc blocks instead.